crates/nu-std/std/mod.nu

```diff
@@ -294,3 +294,35 @@ Startup Time: ($nu.startup-time)
 export def pwd [] {
     $env.PWD
 }
+
+# read from standard input and write to standard output and files
+export def tee [
+    filename: path  # the file to write the input to
+    --append (-a): bool  # append to the file instead of overwriting
+    --force (-f): bool  # force the overwriting of the file's content
+]: any -> any {
+    let data = $in
```

Comment:
This makes the whole thing not streaming based on how `$in` works.

Reply:
What about making it works with coreutils?

Reply:
true, there is a [`tee` command in `uutils`](https://github.com/uutils/coreutils/blob/main/src/uu/tee/tee.md) :open_mouth: